groups/middleware/authorization.go

```diff
@@ -72,13 +87,20 @@ func AuthorizationMiddleware(entityType string, svc groups.Service, repo groups.
 	if err != nil {
 		return nil, err
 	}
+
+	call, err := callout.NewCallout(ctls, certPath, keyPath, caPath, timeout, method, urls, permissions)
```

Comment:
you can init on main.go

Reply:
Yeah I agree, Please init `callout.NewCallout` in main